stacks/java-openliberty-gradle/devfile.yaml

```diff
@@ -18,9 +18,8 @@
 schemaVersion: 2.1.0
 metadata:
   name: java-openliberty-gradle
-  version: 0.4.0
-  displayName: 'Open Liberty Gradle'
-  description: Java application Gradle-built stack using the Open Liberty runtime
+  displayName: Open Liberty Gradle
+  description: Java application based on Gradle using the Open Liberty runtime 22.0.0.1
```

Comment:
add jvm version

Reply:
I'm not sure how to fetch this information. Inside the linked repo I can fetch the maven.compiler property and I'm not sure if it refers to the same thing.

Reply:
Looking at the Dockerfile, it is using OpenJ9 Java 11 (full-java11-openj9-ubi)